assets/global.js

```diff
@@ -553,6 +564,131 @@ class SliderComponent extends HTMLElement {
 
 customElements.define('slider-component', SliderComponent);
 
+class SlideshowComponent extends SliderComponent {
+  constructor() {
+    super();
+    this.sliderControlWrapper = this.querySelector('.slider-buttons');
+    this.bannerContents = this.slider.querySelectorAll('.banner__content');
+
+    if (!this.sliderControlWrapper) return;
+
+    this.sliderItemsToShow = Array.from(this.sliderItems).filter(element => element.clientWidth > 0);
+    this.sliderLastItem = this.sliderItemsToShow[this.sliderItemsToShow.length - 1];
+    if( this.sliderItemsToShow.length > 0) this.currentPage = 1;
+
+    this.sliderControlLinks = this.sliderControlWrapper.querySelectorAll('.slider-counter__link');
+    this.sliderControlLinksArray = Array.from(this.sliderControlLinks);
```

Comment:
Would it make sense to combine both variables into a single one?
I get that you need the array to use `indexOf`, but do you really need the `NodeList` one specifically?

Reply:
You're right 👍  I removed the `NodeList` 👌